manifests/omnibus_package_repository.pp

```diff
@@ -18,35 +18,30 @@
       $_edition = $manage_upstream_edition
     }
 
-    # ensure the correct edition is used if upstream package repositories are being configured
-    if $_edition != 'disabled'{
-      case $facts['os']['family'] {
-        'Debian': {
-          $_filtered_repository_configuration = {
-            'apt::source' => {
-              "gitlab_official_${_edition}" => {
-                location => "https://packages.gitlab.com/gitlab/gitlab-${_edition}/debian",
-              },
-            },
-          }
-        }
-        'RedHat': {
-          $_filtered_repository_configuration = {
-            'yumrepo' =>  {
-              "gitlab_official_${_edition}" =>  {
-                baseurl => "https://packages.gitlab.com/gitlab/gitlab-${_edition}/el/${facts['os']['release']['major']}/\$basearch",
-              },
-            },
+    if $_edition == 'disabled' {
```

Comment:
This whole batch of code was planned to be removed prior to the 3.0 release. It's only in here so that those who use the master in their environments will have a deprecation period.

This isn't a review, just a comment FYI.  I probably should have left a comment to this effect in the code when this was refactored.